### ISSUE TITLE
Nested module readme pages are broken on Registry

### ISSUE THREAD

**Issue by dukedave** [FormidableLabs/terraform-aws-serverless#42]
Both [xray](https://registry.terraform.io/modules/FormidableLabs/serverless/aws/modules/xray/README.md) and [vpc](https://registry.terraform.io/modules/FormidableLabs/serverless/aws/modules/vpc/README.md) submodules give me a **404 Not Found**.

The code seems correct [per the docs](https://www.terraform.io/docs/modules/index.html#standard-module-structure) ⬇️ , so it might be a registry problem?

>  Nested modules should exist under the modules/ subdirectory. Any nested module with a README.md is considered usable by an external user.

**Comment by ryan-roemer** [comment]
Yeah, the hard thing I don't have a solution for is links in those sub-readmes that work on both GitHub _and_ the TF registry.

**Comment by dukedave** [comment]
I think it's gotta be a registry bug, but [I don't see one open](https://github.com/hashicorp/terraform/issues?q=is%3Aissue+label%3Aregistry+is%3Aopen)? 🤔

**Comment by ryan-roemer** [comment]
Here's a bug filed for registry relative link breakage issue: https://github.com/hashicorp/terraform/issues/20930

It was actually the first thing I noticed after publishing this module the first time :)
